packages/ndla-ui/src/Embed/conceptComponents.tsx

```diff
@@ -181,6 +178,9 @@ const StyledList = styled.ul`
     padding: ${spacing.xxsmall};
   }
 `;
+const BylineWrapper = styled.div`
```

Comment:
Hvorfor trenger vi en wrapper her? Kan ikke `EmbedByline` rendres likt alle steder lenger?

Reply:
EmbedByline har ikke noe inline padding lenger, men i concepts er den inn i en modal og da må det legges til litt padding på høyre og venstre side for å matche modaldesignet
![image](https://github.com/NDLANO/frontend-packages/assets/110535103/0d90d46f-675c-46a4-bd4f-1b2e7cc1b9b4)